python/pyfunc-server/docker/local.Dockerfile

```diff
@@ -12,15 +12,25 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-ARG BASE_IMAGE
-
+ARG BASE_IMAGE=ghcr.io/caraml-dev/merlin/merlin-pyfunc-base:0.38.1
```

Comment:
do we have `latest` tag? would be better to use that here so it doesn't require frequent update

Reply:
No, we don't have `latest` tag for all our Merlin components. I agree on having `latest` tag here, but modifying our GitHub actions workflows requires more effort as it currently publishes the docker image on every PR, not only on the release tag. I'll created a separate GitHub issue to track this.

Reply:
Issue created: https://github.com/caraml-dev/merlin/issues/516